### DISCUSSION TITLE
[Pathway Discussion]: WP5424 - How to make the map overall clearer and better organized?

### DISCUSSION THREAD

**Discussion by JuliaBatUM** [wikipathways/wikipathways-help#83] [pathway-discussion]
### Discussion

Hi!
My project goal is to represent oxylipids and their link to senescence on a map. However, as I represented  many reactions, the map is getting quite unclear:
Picture A:
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/690d97e4-75d8-42ee-ae30-1864b4897535)
I think you have a better picture of the map via this link : https://classic.wikipathways.org/index.php/Pathway:WP5424
I thought of representing only some precursors of oxylipids connected to pathways. By clicking on one of these pathways, the production of metabolite Y from its precursor X  is then shown. In the picture below, I represented the lower part of the WP5424 map by adding only the precursors going from alpha-linolenic acid to docosahexaenoic acid. These precursors are connected to pathways (in green). My idea was then, if you would click, for example, on the pathway resolvin T, it would then show a pathway going from docosapentaenoic acid (the precursor) to resolvin T (the downstream metabolite). 
Picture B:
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/3bed653b-8523-4f02-8fde-6b517bfa6b1c)
Unfortunately, if we would represent the map like that, it would not be possible to see all the links in senescence. It would also be more difficult to update changes in the map.
What do you think? Is the map WP5424 clear enough? Or should I maybe still design a map similar to picture B as an overview of the actual bigger map, eg, it might help to identify easier a specific metabolite within the big map WP5424?

Kind regards,
Julia

### Type of revision

- [ ] Major
- [ ] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
Hi @JuliaBatUM. Great description of the problem and possible solutions. I agree with your assessment that replacing whole arms of the pathway with pathway nodes is not ideal, for the reasons you mentioned, but also because the interactivity of pathway nodes is limited. In the classic interface, these nodes are clickable and will open in a new window, but on the new site the individual nodes in the diagram are not clickable in the same way. 

A couple of suggestions:

1. It looks like the lower part of the pathway is not connected to the rest of the pathway, and can perhaps be extracted as a separate pathway representing serie-3-prostaglandins. 
2. To make the pathway less busy, you could do a few things:
      - Minimize duplicated nodes to reduce clutter and help with layout. This doesn't affect data analysis (i.e. enrichment analysis) or data visualization. For example, ALOX5 and ALOX15 are represented multiple times in the lower part of the pathway, and some of these can probably be consolidated by changing the layout slightly.
      - Use shorter labels for metabolites where possible, to make the nodes smaller. For example **2,3-Dinor-Thromboxane B2** => **2,3-Dinor-TXB2**; **4S,5-epoxy-17S-hydroxy-HDHA** => **4S(5)-epoxy-17(S)-HDHA**. Or alternatively, add a line break to longer labels so that the node width can be decreased, which will help with layout.
      - For the **Unknown** nodes, you could remove those nodes and update the affected interaction to a different color, which can be described in a legend (and the description) as "reaction mediated by an unknown enzyme" or similar. Im curious to see what others think of that idea @DeniseSl22 @danidi @egonw 
      - There are also many nodes currently drawn as catalyzing a reaction, but actually labeled as a process, for example Reduction, Hydrolysis, Dehydration, Enzymatic Epoxidation etc. I suggest removing these nodes and instead using interaction color, or maybe a label, to indicate the reaction type.

**Reply by Chris-Evelo** [reply]
Hi both, for bullet 3. I think it is important to discriminate reactions with unknown enzymes (where known ones usually will have a protein ID) and unknown reactions (where known ones can have a reaction ID like KEGG (and maybe RHEA?)). If we do have a reaction ID then I think it would make sense to add that ID to reactions with unknown enzymes.

**Reply by DeniseSl22** [reply]
@khanspers @Chris-Evelo @JuliaBatUM : adding a different color on a line/interaction might complicate downstream analysis. What I'm doing now for lipid pathways, is having a ticker line (e.g. width 2 iso 1) to indicate which interactions are missing a Rhea ID. You could also add a dotted line, to represent an uncertainty (although sometimes people use this to indicate multi-step processes). The key here, is to add a very clear legend. An example of a legend can be found [here](https://classic.wikipathways.org/index.php/Pathway:WP4595). note that these legend are created manually, using 'graphical lines/shapes' (otherwise this will count for unconnected lines or unannotated nodes).

@JuliaBatUM : 
- If you have a pathway with multiple (connected) subpathways, you could use a graphical block to split these up visually, see for example [WP4545](https://classic.wikipathways.org/index.php/Pathway:WP4545) (this is not perfect either, but people can see that there are different elements which are "grouped" together). 
- Having a metapathways like WP5424 is not a bad thing; but the updating of the subpathways might be complicated. Also, if we have the full pathway, and the subpathway, and the metapathway, what will that mean for the pathway statistics @mkutmon ? I believe @marvinm2 is also working on something similar regarding AOPs (see example [WP3980](https://classic.wikipathways.org/index.php/Pathway:WP3980)). There are ways to then link data to these metapathways in Cytoscape, I believe by using the CyTargetLinker?

**Reply by khanspers** [reply]
Just to clarify my comments for point 3 (enzymes without identifiers, labeled Unknown), this is not generally a problem and many pathways have nodes with incomplete annotation where this information is not available. My suggestion to remove these was for this particular pathway, to help make the pathway less busy by reducing the number of nodes (similar to point 4 in the bullet list above). The actual interaction can have an interaction ID or not (Rhea, KEGG), independently of the Unknown enzyme node. And the interaction can be updated with a different line thickness or color, specified in a legend.

**Comment by JuliaBatUM** [comment]
Hi!
Thank you very much for your help!
@khanspers 
I will discuss point 1 with my supervisor.
@Chris-Evelo 
Concerning the reaction IDs, does every reaction (known or unknown) need an ID from KEGG or RHEA, or only the ones where the enzyme is not known?
@DeniseSl22 Indeed, segmenting graphically the pathway would be a good idea. Also, thanks for the legend example.
